src/Okta.Sdk/Internal/DefaultHttpClient.cs

```diff
@@ -42,10 +45,8 @@ public static class DefaultHttpClient
             {
                 Timeout = TimeSpan.FromSeconds(connectionTimeout ?? OktaClientConfiguration.DefaultConnectionTimeout),
             };
-            logger.LogTrace($"Using timeout of {client.Timeout} second(s) from configuration");
 
-            // Workaround for https://github.com/dotnet/corefx/issues/11224
-            client.DefaultRequestHeaders.Add("Connection", "close");
```

Comment:
Do we have a bug tracking the fact that we need to come up with some other workaround to ensure that the `HttpClient` in the SDK switches to the new server in a timely manner when DNS is updated for a failover?

Also, my reading of [the .NET Core thread on HttpClient failover](https://github.com/dotnet/corefx/issues/11224) is that the `connection: close` header should be sent by the *server*, not the client. I think what Darrell Miller is suggesting is that Okta's API servers in the failed region could begin issuing `connection: close` headers to all clients during a failover to ensure that the clients switch to the new endpoint. In other words, the server can close the HTTP connection instead of assuming that the client will be well behaved.

So that might explain why sending `connection: close` from the client is breaking things. 😄

Reply:
I've created an issue on GitHub so we can prioritize later. Thanks @bradhaverstein-okta!